modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties

```diff
@@ -120,9 +132,16 @@ request-string=Request String
 response-string=Response String
 scope-parameter-name=Scope Parameter Name
 search-...=Search...
+search-container=Search Container
+search-container-help=Access helper methods to retrieve Search Container values like the total results count.
 search-insights-help=When a search is performed, this widget displays the query and response strings that are sent to and returned from the search engine. Use this widget for testing search behavior.
 search-options-help=Configure additional search options in this page.
 search-web-configuration-name=Search Web
+search-results-display-context=Search Results Display Context
+search-results-display-context-help=Access helper methods to retrieve the Search Results widget configuration.
```

Comment:
Not sure this is intuitive. What's returned here? The configuration object with all the current values populated?

Reply:
Yes, it's current configuration values for the widget. The most helpful will be `getKeywords` which gets the current search query.

Reply:
How would it be used in the FreeMarker?

I'd like to lead the template developer to this useful information, if possible. Conceptually it sounds like

> Retrieve the Search Results widget's configuration values (for example, getKeywords, which returns the current search query.

But I'm not sure I'm totally clear on it.

Reply:
Yup, it would be used like `searchResultsPortletDisplayContext.getKeywords()`. Clicking on "Search Results Display Context" in the sidebar would output the variable: `searchResultsPortletDisplayContext`.

So that language key sounds good. 👍

Reply:
I only have one more doubt about this one. Do we really mean "configuration" or is there a better way to describe it? When I first read this I was thinking this would correspond to a configuration interface `SearchResultsConfiguration.java` but that doesn't seem to be the case. If this won't cause confusion in the context of developing a widget template, that's fine, just want to make sure.

Reply:
Hmm ya, "configuration" isn't that correct. It's a general collection of values, both configurations and display values, or anything that's needed in the JSP/template.

Reply:
> Retrieve data contained in the Search Results widget's display context (for example, getKeywords, which returns the current search query).

Is that more accurate?

Reply:
Or, we could be even more explicit in case the template dev wants to dig in and see all of the options (aside: is there autocomplete in our editor?)

> Access the data contained in the `SearchResultsDisplayContext` object (for example, its `getKeywords` returns the current search query).

I didn't look to see if that's actually the class name.

Reply:
That sounds really clear. 👌 I'll update the language key with that.

Thanks!